com.ibm.wala.cast.python.ml.test/source/com/ibm/wala/cast/python/ml/test/TestTensorflowModel.java

```diff
@@ -100,32 +103,36 @@ private void testTf2(String filename, String functionName, int expectedNumberOfT
 		// for each pointer key, tensor variable pair.
 		analysis.forEach(p -> {
 			PointerKey pointerKey = p.fst;
-			LocalPointerKey localPointerKey = (LocalPointerKey) pointerKey;
-
-			// get the call graph node associated with the
-			CGNode node = localPointerKey.getNode();
-
-			// get the method associated with the call graph node.
-			IMethod method = node.getMethod();
-			String methodSignature = method.getSignature();
-
-			// associate the method to the pointer key.
-			methodSignatureToPointerKeys.compute(methodSignature, (k, v) -> {
-				if (v == null)
-					v = new HashSet<>();
-				v.add(localPointerKey);
-				return v;
-			});
-
-			TensorVariable tensorVariable = p.snd;
-
-			// associate the method to the tensor variables.
-			methodSignatureToTensorVariables.compute(methodSignature, (k, v) -> {
-				if (v == null)
-					v = new HashSet<>();
-				v.add(tensorVariable);
-				return v;
-			});
+
+			if (pointerKey instanceof LocalPointerKey) {
+				LocalPointerKey localPointerKey = (LocalPointerKey) pointerKey;
+
+				// get the call graph node associated with the
+				CGNode node = localPointerKey.getNode();
+
+				// get the method associated with the call graph node.
+				IMethod method = node.getMethod();
+				String methodSignature = method.getSignature();
+
+				// associate the method to the pointer key.
+				methodSignatureToPointerKeys.compute(methodSignature, (k, v) -> {
+					if (v == null)
+						v = new HashSet<>();
+					v.add(localPointerKey);
+					return v;
+				});
+
+				TensorVariable tensorVariable = p.snd;
+
+				// associate the method to the tensor variables.
+				methodSignatureToTensorVariables.compute(methodSignature, (k, v) -> {
+					if (v == null)
+						v = new HashSet<>();
+					v.add(tensorVariable);
+					return v;
+				});
+			} else
+				logger.warning(() -> "Encountered: " + pointerKey.getClass());
```

Comment:
Minor nit: I really prefer we always use braces for `if` and `else` blocks.  It's ok for now, but we should add a task to re-format Java source in this repo using Google Java Format and then enforce the formatting via spotless.